openapi.yaml

```diff
@@ -647,29 +647,48 @@ components:
       additionalProperties: false
       required:
         - title
-        - summary
+        - startDate
+        - endDate
         - details
         - reward
+        - payoutType
+        - reviewType
         - creatorId
         - workspaceId
         - fields
       properties:
         title:
           type: string
           maxLength: 255
-        summary:
+        startDate:
           type: string
-          maxLength: 1024
+          format: date-time
+          maxLength: 128
```

Comment:
this is irrelevant I think for the date-time format since ISO dates will always be a specific length

Reply:
Will remove maxLength property